ci/run_tests.sh

```diff
@@ -13,7 +13,7 @@ if [[ "not network" == *"$PATTERN"* ]]; then
 fi
 
 if [[ "$COVERAGE" == "true" ]]; then
-    COVERAGE="-s --cov=pandas --cov-report=xml --cov-append"
+    COVERAGE="-s --cov=pandas --cov-report=xml --cov-append --cov-config=setup.cfg"
```

Comment:
Is getting rid of setup.cfg a pre-requisite to ultimately dropping setuptools?

Reply:
This issue is https://github.com/pytest-dev/pytest-cov/issues/243.

We have double config files, one in the source dir, and one that is installed, so coveragepy and pytest(I workaroudn this with ``--import-mode=importlib``) gets confused.

I'll try to fix in a follow up.